app-ios/tutanota/Sources/tutanota-Bridging-Header.h

```diff
@@ -2,6 +2,9 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
+//#define KYBER_K 4
```

Comment:
do we need these definitions still?

Reply:
No, we pass these in as build parameters now. I will remove them.